index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compactible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="./styles.css" />
+    </head>
+    <body>
+        <header class="header">
+            <h2 class="logo">PETER</h2>
```

Comment:
- [ ] It is required that the logo on the webpage should be wrap in an anchor tag `<a>` with a link to the homepage. This is for three reasons: convention, semantics, and user experience(UX). You can check this [link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info about that. Kindly do that to your logo